client/src/components/Public/Map/AddressDetail/AddressDetail.js

```diff
@@ -49,9 +49,9 @@ class AddressDetail extends React.Component<AddressDetailProps> {
   render = () => (
     <div className="address-detail">
       <div className="address-detail-header" style={{height: DETAILS_HEADER_HEIGHT}}>
-        <Button color="link" onClick={this.props.onClick}>
-          Close detail
-        </Button>
+        <button type="button" className="close" onClick={this.props.onClick}>
```

Comment:
Is there a reason to not use `closeAddressDetail` directly?

Reply:
Seems no, tried to use it directly and it works fine. Gonna refactor it.